### PR TITLE
Update Activate Xbox Dev Mode instructions

### PR DESCRIPTION
**Pull request by dannevesdantas**
The current [Xbox One Developer Mode activation](https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/devkit-activation#activate-developer-mode-on-your-retail-xbox-console) document in Microsoft docs is outdated. The instructions provided are no longer valid, given that Xbox system was updated and the Developer mode activation is now being done through Xbox Settings app instead of the old Dev Mode Activation app.

This PR updates the instructions and images in the [devkit-activation.md](https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/devkit-activation#activate-developer-mode-on-your-retail-xbox-console) file and removes the [devkit-deactivation.md](https://docs.microsoft.com/en-us/windows/uwp/xbox-apps/devkit-deactivation) page, since the deactivation process is no longer required as Dev Mode Activation app is no longer used to disable or enable developer mode. It also cleans old images from the [windows-apps-src\xbox-apps\images](https://github.com/MicrosoftDocs/windows-uwp/tree/docs/windows-apps-src/xbox-apps/images) folder that's no longer used, and updates the windows-apps-src\xbox-apps\index.md file.

> NOTE: The screenshots provided were taken from my personal Xbox console, all images was properly blurred to hide my Xbox hardware ID.

> NOTE: Have also confirmed that any refereces to the deleted files doesn't existis anymore. No broken links.

Questions, comments and feedbacks are welcome!

### CODE REVIEW

**Comment by PRMerger12** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@M-Stahl Could you review the proposed changes to your content? The changes include a file deletion, so if approved, the commits will need to be moved to the private repository to create a redirect and update the TOC. Thanks.

**Comment by PRMerger13** [comment]
@dannevesdantas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dannevesdantas** [comment]
Hi @ktoliver. Do you think it might be proper and polite to try to contact @M-Stahl otherwise via LinkedIn or other social network, since he doesn't seem to be very active here on the public GitHub?

**Comment by ShannonLeavitt** [comment]
@jillre @GrantMeStrength - Would you be able to review the proposed changes?

**Review by tswzv21 — APPROVED** [review]

**Comment by aaron-stebner** [comment]
You can't use Developer Settings to switch to dev mode until the Xbox is already activated for UWP dev mode, so several of the proposed changes in this PR aren't accurate.

You have to use the Xbox Dev Mode app (https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) to activate an Xbox for UWP dev mode.

**Comment by dannevesdantas** [comment]
> You can't use Developer Settings to switch to dev mode until the Xbox is already activated for UWP dev mode, so several of the proposed changes in this PR aren't accurate.
> 
> You have to use the Xbox Dev Mode app (https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) to activate an Xbox for UWP dev mode.

Hi @aaron-stebner!

For sure, you’re right.

The dev mode activation process changed after I opened this PR on May 2, 2021 so, I have now updated it to meet the last Xbox OS version, using the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app as you pointed out.

Thanks for the feedback.

Feel free to analyze the proposed changes, feedbacks are welcome!

**Comment by dannevesdantas** [comment]
> You can't use Developer Settings to switch to dev mode until the Xbox is already activated for UWP dev mode, so several of the proposed changes in this PR aren't accurate.
> 
> You have to use the Xbox Dev Mode app (https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) to activate an Xbox for UWP dev mode.

Hi @aaron-stebner!

For sure, you’re right.

Looks like the dev mode activation process changed after I opened this PR on May 2, 2021 so, I have now updated it to meet the last Xbox OS version, using the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app as you pointed out.

Thanks for the feedback.

Feel free to analyze the proposed changes, feedbacks are welcome!

**Comment by dannevesdantas** [comment]
> You can't use Developer Settings to switch to dev mode until the Xbox is already activated for UWP dev mode, so several of the proposed changes in this PR aren't accurate.
> 
> You have to use the Xbox Dev Mode app (https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) to activate an Xbox for UWP dev mode.

Hi @aaron-stebner!

For sure, you’re right.

Looks like the dev mode activation process changed after I opened this PR on May 2, 2021 so, I have now updated it to meet the last Xbox OS version, using the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app as you pointed out.

Thanks for the feedback.

Feel free to analyze the proposed changes, feedbacks are welcome!

File: uwp/xbox-apps/devkit-activation.md [hunk 1]
```diff
@@ -1,73 +1,73 @@
...
+Xbox retail consoles can have two modes, Retail Mode (1) and Developer Mode (2). In Retail Mode, the console is in its normal state: you can play games and run apps acquired through the Microsoft Store. In Developer Mode, you can develop and test software for the console, but you cannot play retail games or run retail apps.
 
-Developer Mode can be enabled on any retail Xbox console, via the “Retail to Dev Kit conversion” app found in the Xbox store. After Developer Mode is enabled on your retail console, you can switch back and forth between Retail (2a) and Developer Modes (2b).
+Developer Mode can be enabled on any retail Xbox console, via the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app found in the Microsoft Store. After Developer Mode is enabled on your retail console, you can switch back and forth between Retail (2a) and Developer Modes (2b).
```
Comment (aaron-stebner) [inline uwp/xbox-apps/devkit-activation.md]: I'd suggest using this link instead (remove the en-us from the path) - https://www.microsoft.com/p/xbox-dev-mode/9nljhzjrn0f4

File: uwp/xbox-apps/devkit-deactivation.md [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
 
    ![Leaving Developer Mode](images/devkit-deactivation-1.png)
 
 Now deactivate your console by using one of the following methods.
 
-## Deactivate your console using the Dev Mode Activation app
+## Deactivate your console using the Xbox Dev Mode app
 
-The preferred method of deactivating Developer Mode on your console is to use the **Dev Mode Activation** app. 
+The preferred method of deactivating Developer Mode on your console is to use the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app.
```
Comment (aaron-stebner) [inline uwp/xbox-apps/devkit-deactivation.md]: Same thing here - remove en-us from the Xbox Dev Mode Store link

File: uwp/xbox-apps/devkit-deactivation.md [hunk 3]
```diff
@@ -1,43 +1,43 @@
...
    
-2.  Open the Dev Mode Activation app.
+2.  Open the Xbox Dev Mode app.
 
 3.  Select **Deactivate**.
   
     ![Deactivate console](images/deactivation-app.png)
 
-See [Xbox One Developer Mode activation](devkit-activation.md) for more information about the **Dev Mode Activation** app. 
+See [Xbox Developer Mode activation](devkit-activation.md) for more information about the [Xbox Dev Mode](https://www.microsoft.com/en-us/p/xbox-dev-mode/9nljhzjrn0f4) app.
```
Comment (aaron-stebner) [inline uwp/xbox-apps/devkit-deactivation.md]: Same thing here - remove en-us from the Xbox Dev Mode Store link

File: uwp/xbox-apps/devkit-activation.md (hunk 1 above)
Comment (dannevesdantas) [reply uwp/xbox-apps/devkit-activation.md]: For sure. Done.

File: uwp/xbox-apps/devkit-deactivation.md (hunk 2 above)
Comment (dannevesdantas) [reply uwp/xbox-apps/devkit-deactivation.md]: Done.

File: uwp/xbox-apps/devkit-deactivation.md (hunk 3 above)
Comment (dannevesdantas) [reply uwp/xbox-apps/devkit-deactivation.md]: Fixed.

**Comment by dannevesdantas** [comment]
@aaron-stebner the en-us portion of the URL was removed. Thanks for the advice.

**Review by aaron-stebner — COMMENTED** [review]
The changes to the wording all look good to me.

The screenshots all look accurate to me but they appear to be different sizes than the previous ones and I don't know of that type of formatting change will matter to the doc build process or not.

I'd like @KevinAsgari to review the updated screenshots if possible before this PR is submitted.

**Comment by KevinAsgari** [comment]
> The changes to the wording all look good to me.
> 
> The screenshots all look accurate to me but they appear to be different sizes than the previous ones and I don't know of that type of formatting change will matter to the doc build process or not.
> 
> I'd like @KevinAsgari to review the updated screenshots if possible before this PR is submitted.

There is a risk of some image quality loss when viewed on larger screens, but they are (almost) all 800 pixel width, so they should still render fine. It's hard to tell until we actually accept the PR and view the docs as rendered on the Learn site though.

**Review by aaron-stebner — APPROVED** [review]

**Comment by dannevesdantas** [comment]
> The changes to the wording all look good to me.
> 
> The screenshots all look accurate to me but they appear to be different sizes than the previous ones and I don't know of that type of formatting change will matter to the doc build process or not.
> 
> I'd like @KevinAsgari to review the updated screenshots if possible before this PR is submitted.

Sure, I also noticed that some screenshots were in different sizes than the others. I took some time trying to figure out which size was the default and noticed that 800x450 is used in the majority of the images, so I opted to use 800x450 also reducing the excessive file size of some images.

**Comment by KevinAsgari** [comment]
@GrantMeStrength I think we can go ahead and merge this PR in now. I don't have authorization to accept the merge myself.

**Comment by dannevesdantas** [comment]
> > The changes to the wording all look good to me.
> > The screenshots all look accurate to me but they appear to be different sizes than the previous ones and I don't know of that type of formatting change will matter to the doc build process or not.
> > I'd like @KevinAsgari to review the updated screenshots if possible before this PR is submitted.
> 
> There is a risk of some image quality loss when viewed on larger screens, but they are (almost) all 800 pixel width, so they should still render fine. It's hard to tell until we actually accept the PR and view the docs as rendered on the Learn site though.

@KevinAsgari before merging, I have just replaced all the Dev Home screenshots by a larger one. I have now noticed that, as the text in the Dev Home screenshot is smaller compared to the other images, it really may look a bit blurry. So, I have replaced all the Dev Home screenshots by a bigger 720p - 1280x720px equivalent in order to avoid any chance of illegibility issues. Also, to avoid the file size to being too large and eventually affecting the ms learn page load speed, I have also compressed it using https://tinypng.com/ to reduce its file size from 840kb to 213kb without perceived quality loss.

The diff:
image

It looks better now.
https://github.com/dannevesdantas/windows-uwp/blob/docs/uwp/xbox-apps/images/activation-step-13-rs4.png

Feel free to go ahead and merge.
Thanks for the advice about the screenshot quality!

**Pull request merged**